proto/brpb.proto

```diff
@@ -236,6 +237,29 @@ message GCS {
     string credentials_blob = 6;
 }
 
+// AzureBlobStorage storage backend saves files into azure blob storage.
+message AzureBlobStorage {
+    string endpoint = 1;
+    // Alias: container
+    string bucket = 2;
+    // Notice: prefix starts without `/`, otherwise the first directory's name is empty.
+    string prefix = 3;
+    // Alias: access_tier. If empty, set `Hot` as default
```

Comment:
oh i thought we have already agreed that the default access_tier is to be decided later.

```suggestion
    // Alias: access_tier.
```